src/Bridge/Symfony/Bundle/DataPersister/TraceableChainDataPersister.php

```diff
@@ -78,6 +87,10 @@ private function tracePersisters($data, array $context = [])
             }
         }
 
-        return $match;
+        foreach ($this->persisters as $persister) {
```

Comment:
I think the `persistersResponse` is not coherent anymore too.

Reply:
That's why there are 2 loops in the `tracePersisters` method, to keep the `persistersResponse` coherent

Reply:
Seems weird to me. The `persistersResponse` is telling that only one persister matches whereas there are more than one.

Reply:
The `tracePersisters` method originally looped on all persisters to store true/false/null value (depending if it supports or not), and returned the first matched persister. The `persistersResponse` should store an array of all persisters looped. For example, with 5 persisters but only the second supports: `[false, true, null, null, null]`

Now I need to handle multiple persisters (thanks to the LoopDataPersisterInterface), but I need to keep the same behavior for the `persistersResponse` array. So if the 2nd persister supports and loops, and the 4th supports (but not the 3rd), it should result something like: `[false, true, false, true, null]`